<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineeringCalculator</class>
 <widget class="QWidget" name="EngineeringCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="div">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="mul">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="shortcut">
    <string>*</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="AllClean">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>139</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrt">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="variable_input">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>143</y>
     <width>321</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string> x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>376</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>atan</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="cos">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="dot">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>375</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="asin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>asin</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="variable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>375</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
   <property name="shortcut">
    <string>^</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="mod">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="shortcut">
    <string>%</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="tan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>376</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="l_bracket">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>139</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
   <property name="shortcut">
    <string>(</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="log10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>log</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>257</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="r_bracket">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>139</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
   <property name="shortcut">
    <string>)</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="Clean">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>139</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="digit_0">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>375</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Digits</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="expression">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>30</y>
     <width>765</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string> 0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>375</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Operations</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ln">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>198</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="acos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>316</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>acos</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">Functions</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="GetResult">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>375</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_y">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_y">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Plot">
   <property name="geometry">
    <rect>
     <x>1201</x>
     <y>375</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../QCustomPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Digits"/>
  <buttongroup name="Functions"/>
  <buttongroup name="Operations"/>
 </buttongroups>
</ui>
